public/js/components/AtomEdit/AtomEdit.js

```diff
@@ -2,6 +2,8 @@ import React, { PropTypes } from 'react';
 import {CTAEditor} from './CustomEditors/CTAEditor';
 import {RecipeEditor} from './CustomEditors/RecipeEditor';
 import {ExplainerEditor} from './CustomEditors/ExplainerEditor';
+import {StoryQuestionsEditor} from './CustomEditors/StoryQuestionsEditor';
```

Comment:
As these are the only exported things inside their files, is there much point in using `{Module}` over making them the default export?